common/static/common/js/discussion/discussion.js

```diff
@@ -36,7 +36,7 @@
                 this.pages = options.pages || 1;
                 this.current_page = 1;
                 this.sort_preference = options.sort;
-                this.is_commentable_cohorted = options.is_commentable_cohorted;
+                this.is_commentable_divided = options.is_commentable_divided;
```

Comment:
I'm not going to change casing now. We will be doing a lot of changes to these JS files for the discussion UI (we will be moving over these Backbone views and changing them, instead of going to React). We can do that sort of cleanup at that time.